src/api.ts

```diff
@@ -63,10 +63,32 @@ export function makeConsentManagerAPI({
       Promise.resolve(
         handleSetViewState(options?.viewState || 'OptOutDisclosure', auth),
       ),
-    showConsentManager: (options) =>
-      Promise.resolve(
-        handleSetViewState(options?.viewState || 'open', undefined, true),
-      ),
+    // eslint-disable-next-line require-await
+    showConsentManager: async (options) => {
+      if (options?.viewState === ViewState.TCF_EU) {
+        logger.error(
+          'TCF_EU view state is not valid for this user experience. ' +
+            'Please configure your Regional Experience to use this view state and try again.',
+        );
+        return;
+      }
+      if (
+        options?.viewState &&
+        !Object.values(ViewState).includes(options.viewState)
+      ) {
+        logger.error(
+          `${
+            options.viewState
+          } is not a valid view state. Valid view states include ${Object.values(
+            ViewState,
```

Comment:
@linh-transcend actually let's only list the "open" view states and not closed view states like closed/collapsed/hidden

Reply:
![Screenshot 2024-02-08 at 10 53 47 AM](https://github.com/transcend-io/consent-manager-ui/assets/94718676/7b222f33-6c07-4a0f-98a3-ec069c216977)

Reply:
LanguageOptions doesn't seem right either, hmm

Reply:
I think this is the enum to use: https://github.com/transcend-io/airgap.js-types/blob/main/src/enums/viewState.ts#L64

Reply:
Updated!
![Screenshot 2024-02-08 at 11 23 50 AM](https://github.com/transcend-io/consent-manager-ui/assets/94718676/03247273-f94a-4302-b26c-16be4302addf)